sdk/src/types/block/output/nft.rs

```diff
@@ -414,8 +414,30 @@ impl NftOutput {
         unlock: &Unlock,
         context: &mut SemanticValidationContext<'_>,
     ) -> Result<(), TransactionFailureReason> {
+        let slot_index = context
+            .transaction
+            .context_inputs()
+            .iter()
+            .find(|c| c.is_commitment())
+            .map(|c| c.as_commitment().slot_index());
+
+        if slot_index.is_none()
+            && (self.unlock_conditions().timelock().is_some() || self.unlock_conditions().expiration().is_some())
+        {
+            // Missing CommitmentContextInput
+            return Err(TransactionFailureReason::SemanticValidationFailed);
```

Comment:
We have `InvalidCommitmentContextInput`

Reply:
Yeah same as in the other PR I think that's also a wrong error since there is no invalid input